src/spell/modules/spell/spell/src/mailbox.rs

```diff
@@ -50,12 +50,11 @@ pub fn push_mailbox(message: String) -> UnitValue {
 }
 
 #[marine]
-/// Get all messages from the mailbox ordered by timestamp ascending.
+/// Get all messages from the mailbox in FIFO order.
 pub fn get_mailbox() -> GetMailboxResult {
     let result: eyre::Result<Vec<String>> = try {
         let conn = db();
-        let statement =
-            conn.prepare(r#"SELECT message FROM mailbox ORDER BY timestamp ASC, id ASC"#)?;
+        let statement = conn.prepare(r#"SELECT message FROM mailbox ORDER BY id DESC"#)?;
```

Comment:
I don't think you can safely order by autoincremented id. It's not guaranteed to be monotonic, if I remember correctly. Gotta order by timestamp and then by id.

Reply:
It's guaranteed to be monotonic for `AUTOINCREMENT`. It's not guaranteed by primary key ids without this keyword. https://www.sqlite.org/autoinc.html

Reply:
> The behavior implemented by the AUTOINCREMENT keyword is subtly different from the default behavior. With AUTOINCREMENT, rows with automatically selected ROWIDs are guaranteed to have ROWIDs that have never been used before by the same table in the same database. And the automatically generated ROWIDs are guaranteed to be monotonically increasing. These are important properties in certain applications. But if your application does not need these properties, you should probably stay with the default behavior since the use of AUTOINCREMENT requires additional work to be done as each row is inserted and thus causes INSERTs to run a little slower.

Ah, they have fixed it. I wonder, in what version 🤔